src/Button/Button.module.css

```diff
@@ -50,3 +49,11 @@
 .Button--secondary:hover {
   box-shadow: inset 0 0 0 2px var(--brand-color-neutral-emphasisPlus);
 }
+
+.Button--size-medium {
+  padding: 1rem 1.5rem;
```

Comment:
can we update these to use the css variables please? 

```suggestion
  padding: var(--base-size-16) var(--base-size-24);
```

Reply:
I was curious about this. The intial implementation was done with the rem values but I knew we had variables.

Reply:
yep, this component predates the design tokens.. it's the OG component 😁